packages/react-native/test/iosUtils.test.ts

```diff
@@ -0,0 +1,125 @@
+import { 
+  errorMessageFromIosException, 
+  backtraceAndDetailsFromIosException, 
+} from '../src/iosUtils'
+
+/* TODO from Bethany June 2023:
+ * The iosUtils code came from Andrey's original code and I took pains
+ * not to mess with how it works. However, in my testing, I've only found 
+ * backtraces generated using framesFromIOSCallStack. I've never seen an
+ * error reported with primaryBackTraceSource = 'ReactNativeComponentStack'
+ * or primaryBackTraceSource = 'ReactNativeIOSStackTrace'
+ * 
+ * Need to check with Andrey if he remembers how to generate an error that 
```

Comment:
@andreybutov do you happen to remember how to trigger these code paths?

Reply:
@andreybutov Could it be related to this change from #1069:
> No longer pass to built-in error handler after notifying in release build. This removes the duplicate error reporting documented in #1054

Reply:
@subzero10 Yes, perhaps. I'll take a look.

Reply:
@andreybutov Yes, javascript errors are currently just going through the JS handler and not passed to the built-in handler in prod because previously those errors were getting double reported. I'm definitely open to handling this differently and appreciate any thoughts you have on it. 

This question can also be broken out into another github issue so we can make this PR just about testing setup -- I think just getting some basic Jest tests running is enough for one PR! I just didn't want to lose track of it.

Reply:
@BethanyBerkowitz @subzero10 I agree. Let’s get the testing stuff through, and let’s open a new issue for the double-reporting / native handling, and I’ll take a look at that separately.

Reply:
Awesome, thanks! I added https://github.com/honeybadger-io/honeybadger-js/issues/1099 and cleaned up this code comment.

Reply:
Thank you @andreybutov. I assigned #1099 to you 👌.